src/components/with-delegations/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentType, memo } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { Delegatee } from '../../types';
+
+import * as actions from './redux/actions';
+
+export interface Props {
+  delegatees: Delegatee[];
+  delegationActions: typeof actions;
+}
+
+const withDelegations = (Component: ComponentType<any>) => {
+  const WrappedComponent = (props: Props) => <Component {...props} />;
+
+  const mapStateToProps = (state: any) => ({
+    delegatees: state.DelegationsReducer.get('delegatees')?.toJS() ?? null
```

Comment:
Denne skal aldri være null 
```suggestion
    delegatees: state.DelegationsReducer.get('delegatees').toJS()
```